CHANGELOG/CHANGELOG-1.3.md

```diff
@@ -13,6 +13,10 @@ Changelog for the K8ssandra Operator, new PRs should update the `unreleased` sec
 
 When cutting a new release, update the `unreleased` heading to the tag being generated and date, like `## vX.Y.Z - YYYY-MM-DD` and create a new placeholder section for  `unreleased` entries.
 
+## unreleased
+
+* [ENHANCEMENT] [#323](https://github.com/k8ssandra/k8ssandra/issues/323) Use Cassandra internals for JMX authentication
```

Comment:
Should we indicate in any way that the issue is in a different project?

Reply:
IMHO we are fine. That happened before, and is usually a symptom that some issue was logged a long time ago in the k8ssandra repo, but was fixed in the k8ssandra-operator repo since we don't bring new features anymore to the k8ssandra repo. I expect these situations to happen less and less frequently though.